photutils/psf/daogroup.py

```diff
@@ -0,0 +1,129 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Functions for implementing the DAOGROUP algorithm proposed by Stetson 1987.
+"""
```

Comment:
Since this is such a central reference for this work, I would give the full bibliographic information:
Astronomical Society of the Pacific, Publications, (ISSN 0004-6280), vol. 99, March 1987, p. 191-222, you could also include the link to the ADS: http://adsabs.harvard.edu/abs/1987PASP...99..191S